docs/6_xccdf_model.md

```diff
@@ -2322,6 +2357,146 @@ Table 11 shows the possible properties of the *\<xccdf:check\>* element.
 |                 |                 |                 | Section         |
 |                 |                 |                 | 7.2.3.5.2.      |
 +-----------------+-----------------+-----------------+-----------------+
+| media-type      | string          | 0-1             | Applicable in   |
```

Comment:
As per @solind, ZIP and TAR are multi-file archives. If we are going to support these, we'd need additional documentation for these. For simplicity sake, I think we should only support single-file archive types.